src/VoiceKey.cpp

```diff
@@ -864,19 +864,16 @@ double VoiceKey::TestEnergy (
 
    while(len > 0)
       {
-         //Figure out how much to grab
-         auto block = limitSampleBufferSize ( t.GetBestBlockSize(s), len );
```

Comment:
Also incorrect to remove `limitSampleBufferSize` here, though this is experimental code unreachable in the release build.